kpi/management/commands/sync_form_media.py

```diff
@@ -0,0 +1,300 @@
+                    user=user,
+                    content=ContentFile(media_content, name=media_filename),
+                    file_type=AssetFile.FORM_MEDIA,
+                    description='default',
+                )
+
+            sync_data.update({'synced_file': True})
+
+            if not quiet and verbosity == 3:
+                synced_files.append(sync_data)
+
+        if not quiet:
+            kpi_after = asset.asset_files.filter(
+                file_type=AssetFile.FORM_MEDIA,
+                date_deleted__isnull=True,
+            ).count()
+            sync_stats = {
+                'asset_uid': asset.uid,
+                'xform_id_string': asset.deployment.xform_id_string,
+                'xformid': asset_xform_id,
+                'asset_owner__username': asset.owner.username,
+                'kobocat_form_media_count': len(
+                    [m for m in media_files if m['xform'] == asset_xform_id]
+                ),
+                'kpi_form_media_pre_sync_count': kpi_before,
+                'kpi_form_media_post_sync_count': kpi_after,
+                'kpi_form_media_count_difference': kpi_after - kpi_before,
+            }
+            if verbosity == 3:
+                sync_stats['synced_files'] = synced_files
+            if verbosity > 1:
+                sync_stats_all.append(sync_stats)
+
+        assets_selected_count += 1
+
+    stats_out = {
+        'assets_selected_count': assets_selected_count,
+    }
+    if not quiet and sync_stats_all and verbosity > 1:
+        stats_out['sync_stats'] = sync_stats_all
+
+    return stats_out
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--asset-uid',
+            action='store',
+            dest='asset_uid',
+            default=None,
+            help="Sync only a specific asset's form-media",
+        )
+        parser.add_argument(
+            '--username',
+            action='store',
+            dest='username',
+            default=None,
+            help="Sync only a specific user's form-media for assets that they own",
+        )
+        parser.add_argument(
+            "--chunks",
+            default=1000,
+            type=int,
+            help="Update records by batch of `chunks`.",
+        )
+        parser.add_argument(
+            '--quiet',
+            action='store_true',
+            dest='quiet',
+            default=False,
+            help='Do not output status messages',
+        )
+
+    def handle(self, *args, **options):
+        if not settings.KOBOCAT_URL or not settings.KOBOCAT_INTERNAL_URL:
+            raise ImproperlyConfigured(
+                'Both KOBOCAT_URL and KOBOCAT_INTERNAL_URL must be '
+                'configured before using this command'
+            )
+
+        stats = _sync_media_files(**options)
+        if not options['quiet']:
+            sys.stdout.write(json.dumps(stats, indent=4) + '\n')
```

Comment:
It's a management command. 
I'm not sure that `json` is the best format to display the stats. What about just printing each member to the console?